challenges/challenge-cowsay-two/solution2.js

```diff
@@ -5,14 +5,46 @@
 // =================
 
 // 1. Make  a command line interface.
+const readLine = require("node:readline");
+const reader = readLine.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
 
+reader.question("Write your word please: ", word => {
+    cow(word);
+    reader.close;
+});
 // 2. Make supplies for our speech bubble
-
+let topLine = "_";
+let bottomLine = "-";
+let sayingWord = ``;
+let cowAscii = `
```

Comment:
It is better to 
- declare variables used only inside a function as local variables in the function
- declare variables that do not change using `const` (e.g., `cowAscii`)